.evergreen/.evg.yml

```diff
@@ -148,7 +148,7 @@ functions:
       params:
         script: |
           ${PREPARE_SHELL}
-          REQUIRE_API_VERSION=${REQUIRE_API_VERSION} LOAD_BALANCER=${LOAD_BALANCER} MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} STORAGE_ENGINE=${STORAGE_ENGINE} ORCHESTRATION_FILE=${ORCHESTRATION_FILE} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
+          REQUIRE_API_VERSION=${REQUIRE_API_VERSION} LOAD_BALANCER=${LOAD_BALANCER} MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} STORAGE_ENGINE=${STORAGE_ENGINE} ORCHESTRATION_FILE=${ORCHESTRATION_FILE} SKIP_LEGACY_SHELL=${SKIP_LEGACY_SHELL} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
```

Comment:
The performance test distro, linux-rhel-9*, is missing a corresponding MongoDB version 5, causing a failure in downloading the legacy shell. To address this, the SKIP_LEGACY_SHELL flag has been added to `bootstrap mongo-orchestration` function. This prevents the download-mongodb.sh script from prompting to download version 5.0.

You can find the specific line of code [here](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/4be70a5bba0f9ca15d10a41aaa6a0092b398bf3b/.evergreen/download-mongodb.sh#L144).